src/sentry/api/fields/actor.py

```diff
@@ -23,9 +30,16 @@ def from_actor_id(cls, actor_id):
         if actor_id.isdigit():
             return Actor(int(actor_id), User)
 
-        type_name, _, id = actor_id.partition(':')
+        if actor_id.startswith('user:'):
+            return cls(int(actor_id[5:]), User)
```

Comment:
Just a small nit: If you're already adopting using `[5:]` it's faster to check `[:5] == 'user:'` then compared to `startswith`.